package.json

```diff
@@ -71,6 +72,7 @@
     "istanbul-middleware": "0.2.2",
     "mocha": "5.2.0",
     "node-sass": "4.9.4",
+    "npm": "^6.4.1",
```

Comment:
npm (I believe from npm-run-all) is otherwise on 4.X, which doesn't support `npm audit --json`

Reply:
Hmm. I think the right way to do this would be to set the correct minimum npm version in the `engines` section, then maybe also add a `config` to set `engine-strict` to true? Might also want to bump the minimum node version in `engines` to the minimum for npm 6.4.1 to work; it's probably higher than 8.0.0, the current minimum.